tests/test_.py

```
@@ -0,0 +1,232 @@
+        pass
+
+Client = None
+if django and not APIClient:
+    try:
+        from django.test import Client
+    except ImportError:
+        Client = None
+
+# --- Fixtures ---
+@pytest.fixture(scope="module")
+def api_client() -> Any:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `api_client`

(ANN401)